tests/AttributeExtensionTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+declare(strict_types=1);
+
+namespace Zalas\PHPUnit\Globals\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Zalas\PHPUnit\Globals\Attribute\Env;
+use Zalas\PHPUnit\Globals\Attribute\Putenv;
+use Zalas\PHPUnit\Globals\Attribute\Server;
+
+#[Env('APP_ENV', 'test')]
+#[Server('APP_DEBUG', '0')]
+#[Putenv('APP_HOST', 'localhost')]
+class AttributeExtensionTest extends TestCase
+{
+    #[Env('APP_ENV', 'test_foo')]
+    #[Server('APP_DEBUG', '1')]
+    #[Putenv('APP_HOST', 'dev')]
+    public function test_it_reads_global_variables_from_method_annotations()
```

Comment:
All these test method names should probably use "attributes" instead of annotations.